omod/src/main/compass/sass/patientsearch/patientSearchWidget.scss

```diff
@@ -3,10 +3,114 @@
 $rowHighlightBackground: #F26522;
 $green: #51A351;
 
-#patient-search-form {
-  input {
+#patient-search-form input {
+  display: inline;
+  color: #363463;
+  padding: 10px 10px;
```

Comment:
Are all these css changes necessary for this ticket? Or just some of it?

Reply:
most of it but I will check it again and see if there's some I can drop.

Reply:
I was making some changes in the css but realized that someone has already made them. I got merge conflicts and trying to resolve them, I decided to keep the current changes so I dropped mine because most of what I had changed was what was there already except for very few trivial changes that I decided to leave out. I think it was you @dkayiwa . :) Thanks.

Reply:
So are you going to remove those unnecessary changes that you made?

Reply:
@dkayiwa I am not intending to remove this css which I added because it is
necessary and directly affects those tags.

+#patient-search-form input {

+  display: inline;
+  color: #363463;
+  padding: 10px 10px;

Reply:
Are @ssmusoke's requested styling changes atlast satisfied?